pages/national_resources/ie_resources.md

```diff
@@ -57,12 +47,38 @@ national_resources:
 <!---Please focus on resources that are relevant for the whole country for life sciences--->
 
 ## Introduction 
-<!---General RDM considerations for your country, how to deal with RDM on a national level--->
 
-## Funders
+This page gives an overview of data management services provided by ELIXIR-Ireland which can be accessed through the [ELIXIR-Ireland website](https://elixir-ireland.ie). 
+
+
+## Funder guidelines and policies
+
+Irish research funding agencies support Open Science and  require data management plans (DMPs) as part of their application process. Consult the funder’s webpages for guidelines and information about research data management.
```

Comment:
@clearysi it would be helpful to list relevant webpages here, these could be general webpages of funding bodies or specific RDM-focused pages on a funding agency's website

Reply:
Hi @Nazeeefa  I've listed the 3 main funding bodies in Ireland underneath that text. I included links to the DMP documents and policies for each funding agency. Should I also include links to their home pages?

Reply:
Great! Sorry I missed it.